index.html

```diff
@@ -1616,16 +1702,17 @@ <h5 id="Psionic Traits">Psionic Traits</h5>
                </td>
                <td><strong>Notes</strong>
                </td>
-               <td><strong>Req</strong>
+               <td><strong>Requirements</strong>
                </td>
             </tr>
             <tr>
                <td>Charm Person
                </td>
-               <td><a class="condition" data-tooltip-container>Charm</a> a creature within 50ft to treat you as a close friend for 10
-                  minutes. The target must be currently not aggressive towards you. 3 WILL points
-                  to resist. If they resist the effect, and/or when the effect ends, they are
-                  aware something tried to control their thoughts.
+               <td>
+                  You may <a class="condition" data-tooltip-container>charm</a> a creature within 50ft to treat you as a close friend for 10
+                  minutes. The target must be currently not aggressive towards you. 2 WILL points
+                  to resist. If they successfully resist the effect or when the effect ends the target creature is 
+                  vaguely aware something tried to control their thoughts, but may not know the source.
```

Comment:
"vaguely" is very well, vague.  Which doesn't feel useful to an inexperienced DM or an annoying player.  GM patch when maybe, but for now probably would like a lil reword